File: src/LiWiMus.Web/Areas/Artist/Profiles/ArtistProfile.cs
using AutoMapper;
using LiWiMus.Web.Areas.Artist.ViewModels;

namespace LiWiMus.Web.Areas.Artist.Profiles;

public class ArtistProfile : Profile
{
    public ArtistProfile()
    {
        CreateMap<CreateArtistViewModel, Core.Artists.Artist>().ReverseMap();
        CreateMap<UpdateArtistViewModel, Core.Artists.Artist>().ReverseMap();
    }
}

File: src/LiWiMus.API/Areas/Artists/Profiles/Artist.Profile.fs
namespace LiWiMus.API.Areas.Artists.Profiles

open AutoMapper
open LiWiMus.API.Areas.Artists.Dto
open LiWiMus.Core.Artists

type UserProfile() as this =
    inherit Profile()
    do this.CreateMap<Artist, ArtistDtoReadList>()
           .ReverseMap()
       |> ignore
